indico/web/client/js/legacy/libs/timetable/Base.js

```diff
@@ -266,14 +266,10 @@ type(
     },
 
     pdf: function() {
-      if ($('html').data('static-site')) {
```

Comment:
`RHTimetableExportDefaultPDF` still uses the legacy PDF. You can easily test the endpoint via `/event/XXX/timetable/timetable.pdf` so you don't actually need to deal w/ the static site stuff. However, keep this logic here since when viewing a static version of the event we cannot show dialogs w/ configuration for obvious resons.

Reply:
I think the change to this file should still be reverted, since showing the dialog in a fully static page (which has no server-side code ;)) won't do anything useful

Reply:
Sure, reverted this